tests/conftest.py

```diff
@@ -9,10 +9,7 @@
 
 # Large swaths of this test infrastructure is shamelessly stolen from semgrep core test infrastructure
 TESTS_PATH = Path(__file__).parent
-MASKED_KEYS = [
-    "tool.driver.semanticVersion",
-    "results.extra.metavars.*.unique_id.md5sum",
-]
+MASKED_KEYS: List[str] = []
```

Comment:
I don't understand this change...

If we're removing the functionality, shouldn't we just delete the constant and its use?

Reply:
Yeah, I wanted to add snapshot tests soon and wanted to also make it obvious that the existing elements were not used.